code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -90,13 +94,18 @@
 
 /mob/living/simple_animal/bot/medbot/Initialize(mapload, new_skin)
 	. = ..()
-	var/datum/job/doctor/J = new /datum/job/doctor
+	var/datum/job/paramedic/J = new /datum/job/paramedic

```

Comment:
why not cmo? it shouldnt get stuck within medbay rooms like surgery/chem

Reply:
paramedic gives them access to base level entries in other departments.